MAUI/Picker/dealing-with-header-and-footer.md

```diff
@@ -0,0 +1,314 @@
+---
+layout: post
+title: Header and footer with .NET MAUI Picker control | Syncfusion
+description: Learn here all about header and footer with Syncfusion .NET MAUI Picker (SfPicker) control.
+platform: maui
+control: SfPicker
+documentation: ug
+---
+
+# Dealing with Header and Footer in .NET MAUI Picker (SfPicker)
+
+This section explains about the header and footer customization of picker control.
+
+## Enable or disable header
+
+SfPicker allows enabling or disabling the header section by setting  `SfPicker.HeaderView.Height` property grater than 0. Default value of `SfPicker.HeaderView.Height` property is 0.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:sfPicker="clr-namespace:Syncfusion.Maui.Picker;assembly=Syncfusion.Maui.Picker"
+             x:Class="Picker_29.MainPage">
+
+    <sfPicker:SfPicker x:Name="picker">
+        <sfPicker:SfPicker.HeaderView>
+            <sfPicker:PickerHeaderView Height="50"/>
+        </sfPicker:SfPicker.HeaderView>
+    </sfPicker:SfPicker>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    picker.HeaderView.Height= 50;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Header customization
+
+SfPicker allows customizing background, text style.
+
+### Background
+
+Header’s background color can be customized by setting `SfPicker.HeaderView.Background` property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+              xmlns:sfPicker="clr-namespace:Syncfusion.Maui.Picker;assembly=Syncfusion.Maui.Picker"
+              x:Class="Picker_29.MainPage">
+
+      <sfPicker:SfPicker x:Name="picker">
+          <sfPicker:SfPicker.HeaderView>
+              <sfPicker:PickerHeaderView Background="Gray"/>
+          </sfPicker:SfPicker.HeaderView>
+      </sfPicker:SfPicker>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    picker.HeaderView.Background=Brush.Gray;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Header text style
+
+The .NET MAUI Picker control, Header text style such as text color, font size, font family, font attributes can be customize as shown in the following code.
```

Comment:
The .NET MAUI Picker control and header text style such as text color, font size, font family, and font attributes, can be customized as shown in the following code.

Reply:
Content modified